README.md

```diff
@@ -39,10 +46,42 @@ Or install it yourself as:
     $ gem install ar2dto
 
 ## Usage
-TBD
+
+In the following sections we explain some basic usage and the API provided by the gem.
 
 ### Setting up your models
-TBD
+
+In order to use the gem you need to add `has_dto` to your ActiveRecord models. For example:
+```ruby
+class User < ApplicationRecord
+  has_dto
+end
+```
+
+This will dynamically create a class called `UserDTO`, and will add two methods to your ActiveRecord model: `#to_dto` and `.to_dto`.
```

Comment:
Once we have defined what goes into these DTOs and what doesn't, we should make it clear here. It should probably be an ActiveModel object with read-only attributes.